test/integration_test/chaosctl/run.sh

```diff
@@ -44,17 +47,120 @@ spec:
 EOF
 kubectl apply -f delay.yaml
 
-echo "Checking chaosctl function"
+echo "Checking chaosctl logs"
 ./chaosctl logs 1>/dev/null
-status=$(./chaosctl debug -i networkchaos web-show-network-delay | grep "Execute as expected")
-if [[ -z "$status" ]]; then
-    ./chaosctl debug -i networkchaos web-show-network-delay
-    echo "Chaos is not running as expected"
+if [ $? -ne 0 ]; then
+    echo "chaosctl logs failed"
+    code=1
+fi
+
+echo "Checking chaosctl debug networkchaos"
+./chaosctl debug networkchaos web-show-network-delay
+if [ $? -ne 0 ]; then
+    echo "chaosctl debug networkchaos failed"
+    code=1
+fi
+echo "Cleaning up networkchaos"
+kubectl delete -f delay.yaml
+rm delay.yaml
+
+echo "Run httpchaos"
+
+cat <<EOF >delay.yaml
+apiVersion: chaos-mesh.org/v1alpha1
+kind: HTTPChaos
+metadata:
+  name: web-show-http-delay
+spec:
+  mode: one # the mode to run chaos action; supported modes are one/all/fixed/fixed-percent/random-max-percent
+  selector: # pods where to inject chaos actions
+    namespaces:
+      - default
+    labelSelectors:
+      "app": "web-show"  # the label of the pod for chaos injection
+  target: Request
+  port: 8081
+  path: "*"
+  delay: "10ms"
+  duration: "30s" # duration for the injected chaos experiment
+EOF
+kubectl apply -f delay.yaml
+
+echo "Checking chaosctl debug httpchaos"
+./chaosctl debug httpchaos web-show-http-delay
```

Comment:
can we check the output of chaosctl?

Reply:
The output format is human-friendly and hard to read by machine.

Reply:
refer to https://github.com/chaos-mesh/chaos-mesh/blob/21564576a61fb21eaf0ba7e60824231add9310bb/test/integration_test/jvm/run.sh#L78
just make sure the output contains some key information

Reply:
Added some assertion.